src/store/actionCreators.ts

```diff
@@ -0,0 +1,16 @@
+import { LOAD_STATUSES_TYPES, Weather, SET_WEATHER_ACTION } from "../types";
+
+
+export const set_error = () => ({
```

Comment:
В js/ts используем камелкейс нотацию, снейк кейс для констант (которые пишем капслоком и из-за этого камелкейс невозможен)
```suggestion
export const setError = () => ({
```